scripts/gen_server_keys.sh

```diff
@@ -18,21 +18,24 @@ COMMONNAME=""
 ADDRESS=""
 SUBJECT="/C=US"
 
-while getopts d:a: option; do
+while getopts d:a:s: option; do
     case "${option}" in
 	d) COMMONNAME=${OPTARG};;
 	a) ADDRESS=${OPTARG};;
-	*) echo "usage: ./generate.sh -d <domain> -a <ip_address>"; exit 1;;
+	s) SAN_DNS=${OPTARG};;
+	*) echo "usage: ./generate.sh -d <domain> -a <ip_address> -s <san_extension_DNS>"; exit 1;;
     esac
 done
 
 if [[ -n "${COMMONNAME}" ]]; then
     SUBJECT="${SUBJECT}/CN=${COMMONNAME}"
 fi
 
-SANEXT="[SAN]\nbasicConstraints=CA:TRUE\nsubjectAltName=DNS:localhost"
+# TODO(ismail): make the IPs configurable as well
+ALTNAMES="[alt_names]\nDNS.1=${SAN_DNS}\nDNS.2=localhost\nIP.1=0.0.0.0\nIP.2=35.184.134.53"
+SANEXT="[SAN]\nbasicConstraints=CA:TRUE\nsubjectAltName=@alt_names\n\n${ALTNAMES}"
 if [[ -n "${ADDRESS}" ]]; then
-    SANEXT="${SANEXT},IP.2:${ADDRESS}"
+    SANEXT="${SANEXT}"
```

Comment:
The address option here is ignored, which might be confusing for users. I suggest fixing it before merging.